my_vacation.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <title>Korey Goudreau</title>
+        <meta charset="UTF-8">
+        <meta name="description" content="Vacation Webpage">
+        <meta name="keywords" content="HTML, CSS, JavaScript, Vacation">
+        <meta name="author" content="Korey Goudreau">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body style="background-image: url('vacationBackground.jpg');background-repeat: no-repeat;background-attachment: fixed;background-size: 100% 100%;">
+        <a href="index.html">
+            <img src="airplane.jpg" alt="Boarding Airplane to go back Home" title="Time to go back home..." style="width:10%;display:block;margin-left:auto;margin-right:auto;">
```

Comment:
This is really nice! 

But I had to come see the code to discover that the image was a link, it was not clear at first! Maybe you could add a small text above it.